CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Use only unix line-endings in source files
 - Use set instead of list for Converter.extensions
 - Converted datasets are hilbert-curve sorted
+- Reimplement converters to class-based converters
```

Comment:
A new converter, at_crop, snuck in. That should either be PR'ed separately or be part of the changelog.

Reply:
Maybe also mention the addition of multiple years to be_vlg

Reply:
True, I was updating the Austrian "Feldblocke" and realized I had changed it to a new converter (crop fields). I'll add it to the changelog